index.html

```diff
@@ -84,6 +84,9 @@
       license: "w3c-software-doc",
     };
     </script>
+    <style>
+      table#linking-table, table#linking-table th, table#linking-table td { border: 1px solid #000; }
```

Comment:
Is there anything in base.css we could use instead? If no, then I'm ok with merging this.

Reply:
If we are not fixated on that HTML4-era table look, we can use `complex table` class and add `<thead>` and `<tbody>`. I pushed an update with a more modern look using base.css classes.